documentation/documentation/docs/evaluators/python_evaluator.md

```diff
@@ -0,0 +1,13 @@
+# Python evaluator
+## General usage
+This evaluator is responsible for running and executing tests on a student's Python code.
+
+## Structure
+When submitting the project a teacher can add a requirements manifest `req-manifest.txt`, this way only the packages in the requirements file are usable on the evaluator.
+
+When no manifest is present, students are able to install their own depedencies with a `requirements.txt` and a `dev-requirements.txt`.
+Or the teacher can add a `requirements.txt` if they want to control the dependencies that a student can use.
```

Comment:
The `requirements.txt` just installs the dependencies, it doesn't control nothing. Students can still use `requirements.txt`

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/ded58d55e5af39684118954f822d6c1ee3f0db15